### DISCUSSION TITLE
OCaml binding: How to enable parallelism?

### DISCUSSION THREAD

**Discussion by liyishuai** [Z3Prover/z3#7416] [Q&A] [answered]
https://github.com/Z3Prover/z3/blob/00f1f1b83da7c583d0290280b1b3bbb6097a6be8/RELEASE_NOTES.md?plain=1#L512-L514

I tried to call the [`mk_context`](https://z3prover.github.io/api/html/ml/Z3.html#VALmk_context) function with `["parallel.enable", "true"]`, but received:
> WARNING: unknown parameter 'parallel.enable'
Legal parameters are:
  auto_config (bool) (default: true)
  debug_ref_count (bool) (default: false)
  dot_proof_file (string) (default: proof.dot)
  dump_models (bool) (default: false)
  encoding (string) (default: unicode)
  model (bool) (default: true)
  model_validate (bool) (default: false)
  proof (bool) (default: false)
  rlimit (unsigned int) (default: 0)
  smtlib2_compliant (bool) (default: false)
  stats (bool) (default: false)
  timeout (unsigned int) (default: 4294967295)
  trace (bool) (default: false)
  trace_file_name (string) (default: z3.log)
  type_check (bool) (default: true)
  unsat_core (bool) (default: false)
  well_sorted_check (bool) (default: false)

**Comment by NikolajBjorner** [comment, marked as answer]
you can other parameters, such as parallel.enable, as global parameters.

https://z3prover.github.io/api/html/ml/Z3.html#VALset_global_param
